packages/resolver/test/globalnpm.ts

```diff
@@ -5,6 +5,7 @@ import * as sinon from "sinon";
 import path from "path";
 import fs from "fs";
 import { describe, it } from "mocha";
```

Comment:
Oh, huh, what are these doing here?  These don't need to be imported, they're globals... (not that you changed that, sorry, I just noticed that and it seemed off)